### ISSUE TITLE
help!

### ISSUE THREAD

**Issue by Trey13208** [checkra1n/BugTracker#228]
**What device + iOS version are you on?**

iphone 8 ios 13.1
**What checkra1n version are you using?**
.9.2
**What are the steps to reproduce the issue?**
1.
2.
3.
...

**What do you expect, and what is happening instead?**
It wont let me go into the dfu mode when trying to install the jailbreak from my mac i do the steps and my phone simply turns on. Any help?
**Any other info, error logs, screenshots, ...?**

**Comment by networkextension** [comment]
volume down  (left down) key ,and right key as power button

**Comment by Trey13208** [comment]
You mean like the volume up button?

**Comment by perpelator** [comment]
@Trey13208 Let me know if this works. 1. Plug in your device. 2: Click start and get into recovery mode via Checkra1n (don’t manually go into recovery mode via buttons yourself. Let checkra1n do it for you by pressing next). 3: Once your device is detected again, begin to start the DFU mode process. After get ready (which is 3 seconds), the next step will begin at 6 seconds. Do NOT begin until that hits 5 seconds. 4: Continue to do what it says at the normal amount of said time.

Personally I started at 6 seconds but then cut all the other steps short by 1 second, but it’ll work either way. I personally was using a Hackintosh via usb a 2.0 on MacOS 10.13.6. Cannot confirm this method works (but it should) for other versions. I tested this on both 0.9 and 0.9.1, so should carry over with the following versions until fixed. Happy Jailbreaking :)
